mapproxy/cache/base.py

```diff
@@ -42,6 +42,9 @@ class TileCacheBase(object):
 
     supports_timestamp = True
 
+    def __init__(self, coverage=None) -> None:
+        self.coverage = coverage
```

Comment:
Where is the coverage used in this class? I cannot see it, can you give me a point where i can see where the coverage is handled for the different caches?

Reply:
Sure!

The `cache` coverage is accessed in the `loader` class which calculates each caches extent in the `caches` function before return (line 1668):
https://github.com/mapproxy/mapproxy/pull/691/files#diff-7557b8794cb455eb559651f8fa4730d2a8a55d2c13bd188d730eecb4ed10e863R1668

It is also accessed in the `tile` class under cache directory in the `load_tile_coords` function in order to remove tiles that are not in the `coverage` (line 156):
https://github.com/mapproxy/mapproxy/pull/691/files#diff-cc6dc27bf169f009bbbd33027f122d17a5cac2e3f4224db2b58d3869b59673d9R156

Reply:
Thanks